#748 Update info about which K8s distros can enable ACE (#749)
rancher/rancher-docs: versioned_docs/version-2.6/reference-guides/cluster-configuration/rancher-server-configuration/rke1-cluster-configuration.md

### Authorized Cluster Endpoint

Authorized Cluster Endpoint (ACE) can be used to directly access the Kubernetes API server, without requiring communication through Rancher.

:::note

> ACE is only available on Kubernetes clusters launched by Rancher. In Rancher v2.6.3 and later, this means clusters where Rancher used RKE, RKE2, or K3s to provision the cluster. Prior to Rancher v2.6.3, it was only available for RKE. It's also not available for clusters in a hosted Kubernetes provider, such as Amazon's EKS.

Reviewer: ```suggestion
ACE is only available on Kubernetes clusters provisioned by Rancher. In Rancher v2.6.3 and later, RKE, RKE2, and K3s all support ACE. Prior to Rancher v2.6.3, it was only available for RKE. Regardless of version, ACE is not available for clusters in a hosted Kubernetes provider, such as Amazon's EKS.
```

:::

> ACE must be set up [manually](../../../how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/register-existing-clusters#authorized-cluster-endpoint-support-for-rke2-and-k3s-clusters.md) on RKE2 and K3s clusters in Rancher v2.6.3 and later. In RKE, ACE is enabled by default in Rancher-launched Kubernetes clusters, using the IP of the node with the `controlplane` role and the default Kubernetes self signed certificates.

Reviewer 2: ```suggestion
ACE must be set up [manually](../../../how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/register-existing-clusters#authorized-cluster-endpoint-support-for-rke2-and-k3s-clusters.md) on RKE2 and K3s clusters in Rancher v2.6.3 and later. In RKE, ACE is enabled by default in Rancher-launched Kubernetes clusters, using the IP of the node with the `controlplane` role and the default Kubernetes self-signed certificates.
```

Reviewer: Hmm, we link here to the RKE2/K3s docs for registering existing clusters, and talk about enabling ACE on these clusters. It sounds like some of the previous language about only supporting ACE on clusters provisioned is not correct, and it can in fact be enabled on an RKE2 or K3s cluster. Can we verify that?

Author: @aiyengar2 Can you or someone else on the team confirm the above comment?

Reviewer 3: I did a quick check on this and saw that we can register (import) existing RKE2/k3s clusters and was able to enable/disable ACE on both of these cluster types.

Author: From Slack:

> AFAIK it still will not work for hosted providers. We have special integrations for k3s/rke2 as those are our distros
